autometa/common/markers.py

```diff
@@ -201,21 +201,22 @@ def get(self, format='wide', **kwargs):
                 orfs=self.orfs_fpath)
         return Markers.load(fpath=self.markers_fp, format=format)
 
-def main(args):
-    markers = Markers(orfs_fpath=args.orfs, kingdom=args.kingdom, dbdir=args.dbdir)
-    markers.get()
-
-if __name__ == '__main__':
+def main():
     import argparse
     import logging as logger
     logger.basicConfig(
         format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p',
         level=logger.DEBUG)
-    parser = argparse.ArgumentParser('')
+    parser = argparse.ArgumentParser(description='Annotate ORFs with kingdom-marker information')
     parser.add_argument('orfs', help='</path/to/prot.orfs.faa>')
     parser.add_argument('kingdom', help='kingdom to search for markers',
         choices=['bacteria','archaea'], default='bacteria')
-    parser.add_argument('--dbdir', help='</path/to/markers/dir>', default=MARKERS_DIR)
+    parser.add_argument('--dbdir', help=f'</path/to/markers/dir> {MARKERS_DIR}',
+        default=MARKERS_DIR)
```

Comment:
As the user might run this script standalone, the help messages have to be very clear. Here, it should be made clear what we mean. For example "Path to directory containing the single-copy marker HMM databases".